app/views/teachers/show.html.erb

```diff
@@ -1,10 +1,99 @@
-<h1>Teacher <%= @teacher.to_s %></h1>
-<% if session[:current_lecture] %>
-  <%= button_to "Current lecture", lecture_path, method: :get, id: session[:current_lecture][:id] %>
-<% end %>
-<p>
-  <strong>bella pe te</strong>
-</p>
-<%= button_to "New lecture", new_lecture_path, method: :get, params: { teacher_id: @teacher.id } %>
-
-<%= link_to 'Back', teachers_path %>
\ No newline at end of file
+<div class="sidebar">
+  <%= link_to "Home", root_path, :class => 'active' %>
+  <%= link_to "Courses", school_class_path %>
+  <% for school_class in @teacher.school_classes %>
+    <%= link_to school_class.number.to_s + school_class.section.to_s, school_class_path(school_class) %>
+  <% end %>
+  <%= link_to "Classes", school_class_path %>
+  <%= link_to "Today's lectures", lecture_path %>
+</div>
+
+<div class="content">
+  <h1>Welcome back, <%= @teacher.to_s %></h1>
+  <% if session[:current_lecture] %>
+    <%= button_to "Current lecture", lecture_path, method: :get, id: session[:current_lecture][:id] %>
+  <% end %>
+
+  <!-- TO BE COPIED IN THE SCHOOL CLASS PAGE RELATIVE TO THIS TEACHER
+  <div style="margin-left: 30px">
+    <h3>Add a new lecture</h3>
+    <div>
+    <% for subject in  @teacher.subjects %>
+      <%= button_to subject, new_lecture_path(@teacher), :class => 'btn btn-primary' %>
+    <% end %>
+    </div>
+  </div>
+  -->
+</div>
+
+<style>
+  /* The side navigation menu */
+  .sidebar {
```

Comment:
Il css nei file del css, non in mezzo alle viste

Reply:
Sempre per questioni di visibilità

Reply:
Tolto